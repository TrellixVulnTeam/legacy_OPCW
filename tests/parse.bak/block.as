def f() -> int { { return 3; } }

{ def main() { } }

{ { { if false {  } }}}
