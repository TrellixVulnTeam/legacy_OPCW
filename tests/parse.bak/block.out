ModuleDecl <?> _
  Nodes <?> 
    FuncDecl <?> f
      Ident <?> int
      Parameters <?> 
      Nodes <?> 
        Block <?>
          Nodes <?> 
            ReturnExpr <?> 
              IntegerExpr <?> 3 (10)
    Block <?>
      Nodes <?> 
        FuncDecl <?> main
          Parameters <?> 
          Nodes <?> 
    Block <?>
      Nodes <?> 
        Block <?>
          Nodes <?> 
            Block <?>
              Nodes <?> 
                SelectExpr <?> 
                  Branches <?> 
                    SelectBranch <?> 
                      BooleanExpr <?> false
                      Nodes <?> 
